projects/xtoken.js

```diff
@@ -1,9 +1,16 @@
+const env = require("dotenv").config();
 const retry = require("async-retry");
 const axios = require("axios");
 const utils = require("./helper/utils");
-const web3 = require('./config/web3.js');
+const Web3 = require("web3");
+const web3 = new Web3(
+  new Web3.providers.HttpProvider(
+    `https://mainnet.infura.io/v3/${env.parsed.INFURA_KEY}`
```

Comment:
What's the reasoning behind using this instead of config/web3.js ? Is it because testing is easier?

Reply:
Yes, apologies I have been using that pattern locally to test. I amended to file to maintain the config/web3.js implementation.

Reply:
Got it :+1: